--- kudzu-1.1.40/ddcprobe/lrmi.c	2003-02-11 15:47:38.000000000 +0100
+++ kudzu-1.2.24/ddcprobe/lrmi.c	2005-12-14 18:27:31.000000000 +0100
@@ -792,15 +792,15 @@
 	{
 	unsigned int vret;
 	sigset_t allsigs, cursigs;
-	unsigned long oldgs;
+	unsigned short oldgs;
 
 	while (1)
 		{
 		sigfillset(&allsigs);
 	        sigprocmask(SIG_SETMASK, &allsigs, &cursigs);
-		asm volatile ("movl %%gs, %0" : "=g" (oldgs));
+		asm volatile ("mov %%gs, %0" : "=g" (oldgs));
 		vret = lrmi_vm86(&context.vm);
-		asm volatile ("movl %0, %%gs" :: "g" (oldgs));
+		asm volatile ("mov %0, %%gs" :: "g" (oldgs));
 		sigprocmask(SIG_SETMASK, &cursigs, NULL);
 
 		if (VM86_TYPE(vret) == VM86_INTx)
